cmd/afsPmf/afsPmf.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/popgen"
+	"log"
+)
+
+func afsPmf(vcfFile string, outFile string, unPolarized bool) {
+	g, err := popgen.VcfToAfs(vcfFile, !unPolarized) //VcfToAfs is written in terms of polarized, so this is inverted here.
+	exception.FatalOnErr(err)
+	out := fileio.EasyCreate(outFile)
+
+	numAlleles := g.Sites[0].N
```

Comment:
assumption of N being constant for all sites?

Reply:
yes, as we are presenting a probability mass function, which requires constant n